### PR TITLE
PRESS-2 761 - Implement Interface Skeleton

### PR DESCRIPTION
**Pull request by officiallygod**

### CODE REVIEW

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+			{ ...navigateRegionsProps }
+			ref={ useMergeRefs( [ ref, navigateRegionsProps.ref ] ) }
+			className={ classnames(
+				className,
+				'nfd-interface-interface-skeleton',
+				navigateRegionsProps.className,
+				!! footer && 'has-footer'
+			) }
+		>
+			{ !! drawer && (
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js:82]: Can we just check why `!!` was used?

**Review by arunshenoy99 — COMMENTED** [review]
Did not do an in-depth check since this code is from Gutenberg, suggested a minor change that we might have missed when pasting. Rest all looks good.

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js (hunk 1 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js:82]: The double exclamation point, or double bang, converts a truthy or falsy value to “true” or “false”. In other words, it operates exactly like Boolean(value) would. [Ref](https://medium.com/@chirag.viradiya_30404/whats-the-double-exclamation-sign-for-in-javascript-d93ed5ad8491#:~:text=The%20double%20exclamation%20point%2C%20or,like%20Boolean(value)%20would.)

Comment (arunshenoy99) [reply src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js:82]: Thanks

**Review by arunshenoy99 — APPROVED** [review]

**Pull request merged**